orchestration/tests/test_cli_support.py

```
@@ -201,6 +202,102 @@ def which_side_effect(command):
+                self.dispatcher,
+                "_create_worktree_at_location",
+                return_value=("/tmp/task-agent-cli-args-test", MagicMock(returncode=0, stderr="")),
+            ),
+            patch("os.makedirs"),
+            patch("os.chmod"),
+            patch("builtins.open", mock_open()),
+            patch("os.path.exists", return_value=False),
+            patch("orchestration.task_dispatcher.Path.write_text") as mock_write_text,
+            patch("orchestration.task_dispatcher.subprocess.run") as mock_run,
+            patch("orchestration.task_dispatcher.shutil.which") as mock_which,
+            patch.object(self.dispatcher, "_validate_cli_availability", return_value=True) as mock_validate,
```

Ruff (0.15.2)

[error] Local variable `mock_validate` is assigned to but never used

Remove assignment to unused variable `mock_validate`

(F841)